modules/swagger-models/src/main/java/io/swagger/models/parameters/AbstractSerializableParameter.java

```diff
@@ -169,7 +170,7 @@ public void setProperty(Property property) {
         this.format = property.getFormat();
         if (property instanceof StringProperty) {
             final StringProperty string = (StringProperty) property;
-            setEnum(string.getEnum());
+            if (string.getEnum() != null) setEnum(new ArrayList<Object>(string.getEnum()));
```

Comment:
The `enum` property from other primitive types has to be taken into account as well (`IntegerProperty`, `LongProperty`, `FloatProperty`, `DoubleProperty`, etc.)